components/AlertNotification/AlertNotification.test.tsx

```diff
@@ -30,7 +31,36 @@ const variantTestCases = {
 describe('AlertNotification', () => {
   for (const [key, value] of Object.entries(variantTestCases)) {
     it(`renders the correct variant ${key}`, () => {
-      render(<Alert variant={AlertVariants[key]} message={value.message} />);
+      render(
+        <Alert
+          variant={AlertVariants[key as keyof typeof AlertVariants]}
+          message={value.message}
+        />,
+      );
     });
-  }
+
+    it('should render the dismiss button on each alert type', () => {
+      render(
+        <Alert
+          variant={AlertVariants[key as keyof typeof AlertVariants]}
+          message={value.message}
+        />,
+      );
+      const dismissButton = screen.getByTestId('dismiss-alert-btn');
+      expect(dismissButton).toBeInTheDocument();
+    });
+
+    test('should fire the toast.remove() function when dismiss button is clicked', async () => {
```

Comment:
We're using it instead of test.
```suggestion
    it('should fire the toast.remove() function when dismiss button is clicked', async () => {
```

Reply:
I still don't see that this updated to `it`.